code/pcgen.sh

```diff
@@ -1,6 +1,20 @@
 #!/bin/sh
 set -e
 
+if [ ! -e ./pcgen.sh ]; then
```

Comment:
Can you add a comment explaining the Mac OS X use case here?

Reply:
Done.